src/data/apiMetadata.js

```diff
@@ -436,263 +841,160 @@ export default [
-        units: "People",
-      },
-      {
         categories: [
-          { code: "KS102EW0002", name: "Age 0 to 4", slug: "age-0-to-4" },
-          { code: "KS102EW0003", name: "Age 5 to 7", slug: "age-5-to-7" },
-          { code: "KS102EW0004", name: "Age 8 to 9", slug: "age-8-to-9" },
-          { code: "KS102EW0005", name: "Age 10 to 14", slug: "age-10-to-14" },
-          { code: "KS102EW0006", name: "Age 15", slug: "age-15" },
-          { code: "KS102EW0007", name: "Age 16 to 17", slug: "age-16-to-17" },
-          { code: "KS102EW0008", name: "Age 18 to 19", slug: "age-18-to-19" },
-          { code: "KS102EW0009", name: "Age 20 to 24", slug: "age-20-to-24" },
-          { code: "KS102EW0010", name: "Age 25 to 29", slug: "age-25-to-29" },
-          { code: "KS102EW0011", name: "Age 30 to 44", slug: "age-30-to-44" },
-          { code: "KS102EW0012", name: "Age 45 to 59", slug: "age-45-to-59" },
-          { code: "KS102EW0013", name: "Age 60 to 64", slug: "age-60-to-64" },
-          { code: "KS102EW0014", name: "Age 65 to 74", slug: "age-65-to-74" },
-          { code: "KS102EW0015", name: "Age 75 to 84", slug: "age-75-to-84" },
-          { code: "KS102EW0016", name: "Age 85 to 89", slug: "age-85-to-89" },
-          { code: "KS102EW0017", name: "Age 90 and over", slug: "age-90-and-over" },
+          {
+            code: "QS104EW0002",
+            name: "Female",
+            slug: "female",
+          },
+          {
+            code: "QS104EW0003",
+            name: "Male",
+            slug: "male",
+          },
         ],
-        code: "KS102EW",
-        name: "Age",
-        slug: "age",
-        desc: "The age of a person on Census Day, 21 March 2021.",
-        total: { code: "KS102EW0001", name: "All categories: Age", slug: "all-categories-age" },
         units: "People",
+        total: {
+          code: "QS104EW0001",
+          name: "",
+          slug: "",
+        },
       },
       {
+        code: "KS103EW",
+        name: "Marital status",
+        slug: "marital-status",
+        desc: "People married of in civil partnerships.",
         categories: [
           {
             code: "KS103EW0002",
-            name: "Single",
-            slug: "single",
+            name: "Single never married or in a civil partnership",
+            slug: "single-never-married-or-in-a-civil-partnership",
           },
           {
             code: "KS103EW0003",
             name: "Married",
             slug: "married",
           },
+          {
+            code: "KS103EW0004",
+            name: "In a registered civil partnership",
+            slug: "in-a-registered-civil-partnership",
+          },
           {
             code: "KS103EW0005",
-            name: "Separated",
-            slug: "separated",
+            name: "Separated (but still legally married)",
+            slug: "separated-but-still-legally-married",
```

Comment:
How can you be illegally married?

Reply:
Apparently so - forced marriages and so on